src/intro-to-curve-dodao-academy.yaml

```diff
@@ -145,31 +145,31 @@ steps:
           - 52c4c10a
           - 31cbd68f
         choices:
-          - content: To earn rewards through bribing
+          - content: To earn new tokens for veCRV wotes in bribing
             key: ad0cd11f
           - content: To get voting power in gauge weight
             key: c38ae3e6
           - content: Participate in Curve DAO governance
             key: 52c4c10a
-          - content: Gain more rewards by investing veCRV in Convex Finance
+          - content: To earn new tokens for veCRV wotes in bribing
```

Comment:
The first and last option is the same. You can make it a single choice question with the last choice as "All the above"

Reply:
ok sir